packages/forms/src/model.ts

```diff
@@ -1450,22 +1450,36 @@ export class FormGroup extends AbstractControl {
    *
    * @param name The control name to add to the collection
    * @param control Provides the control for the given name
+   * @param options Allows to specify whether this FormGroup instance should emit events after a new
```

Comment:
As an alternative suggestion here:
```suggestion
   * @param options Allows the ability to specify whether this FormGroup instance should emit events after a new
```

Reply:
Or possibly:
```suggestion
   * @param options Specifies whether this FormGroup instance should emit events after a new
```